lib/stdlib/fields.juttle

```diff
@@ -0,0 +1,74 @@
+// Manipulate fields within a stream with these field reducers
+
+// Return an array of last N values of a field
+//
+// Parameters:
+//
+//   * field: ("value") field to tail
+//   * length: (1) number of recent values to retain
+export reducer tail(field, length=1) {
+    var data = [];
+
+    function update() {
+        if (*field != null) { 
```

Comment:
No, we don't have a way to tell the two apart. To me, whether they should or shouldn't count is unclear...but if its clear to others I'm game.